### PR TITLE
Adds test for Stacked Bandwidths

### PR DESCRIPTION
**Pull request by r-akemii**
In this PR we test the functions `DesireLinesDialog`, `DesireLinesProcedure`, `LoadColorRampSelector`, and `CreateBandwidthsDialog` from the GIS module, increasing the module coverage for visualization tools.

We also removed a code line that printed the result for the Delaunay line procedure and added a new test fixture that creates arbitrary desirelines for test.

### CODE REVIEW

**Comment by r-akemii** [comment]
@pedrocamargo with the addition of these new tests, we should reach 70% test coverage for QAequilibraE code. We're still missing tests for `SimpleTagDialog` and `LeastCommonDenominatorDialog` in the GIS module. I'm not sure what to do to test these functions, but I'll check with you later.

**Review by pedrocamargo — CHANGES_REQUESTED** [review]
Remote the use of desire lines. Add layer from test data (create some for that)

**Comment by r-akemii** [comment]
@pedrocamargo I added a fixture in *conftest* to create an arbitrary line layer to test bandwidth creation, and the code piece that made the desirelines was tested separately in a new test file.

BTW, we have 72% coverage with the tests in this branch.

File: qaequilibrae/modules/gis/desire_lines_procedure.py [hunk 1]
```diff
@@ -270,7 +270,7 @@ def do_delaunay_lines(self):
         assigner = allOrNothing(self.matrix, self.graph, self.results)
         assigner.execute()
         self.report = assigner.report
-        print(self.results.link_loads)
+        # print(self.results.link_loads)
```
Comment (pedrocamargo) [inline qaequilibrae/modules/gis/desire_lines_procedure.py]: remove this

File: test/test_create_desirelines.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    # Test if Desireline was indeed created
+    prj_layers = [lyr.name() for lyr in QgsProject.instance().mapLayers().values()]
+    assert line_layer in prj_layers
+
+    layer = QgsProject.instance().mapLayersByName(line_layer)[0]
+    if is_delaunay:
+        assert layer.featureCount() == 62
+    else:
+        assert layer.featureCount() == 264
```
Comment (pedrocamargo) [inline test/test_create_desirelines.py]:
```suggestion
    target = 62 if is_delaunay else 264
    assert layer.featureCount() == target
```

**Review by pedrocamargo — APPROVED** [review]

**Pull request merged**